guides/doc-Deploying_on_AWS/topics/AWS_Use_Case_Considerations.adoc

```diff
@@ -50,7 +50,7 @@ You must do this when {ProjectServer} or {SmartProxyServer} has different intern
 
 ifdef::katello,orcharhino,satellite[]
 .On demand content sources
-You can use the *On demand* download policy to reduce the storage footprint of the Red Hat Enterprise Linux server that runs {Project}.
+You can use the *On demand* download policy to reduce the storage footprint of the {RHEL} server that runs {Project}.
```

Comment:
Use `RHELServer` attribute.

Reply:
No, this is explicitly not `RHELServer`. RHELServer is a product. This refers to a server that runs RHEL. Note the lowercase s. I also considered making it
```suggestion
You can use the *On demand* download policy to reduce the storage footprint of the server that runs {Project}.
```
Which IMHO relays just as much, but I thought to keep the scope minimal.

Reply:
I like your suggestions. Fine by me if you squash this into this PR/commit.